rust/migrate-wicked/tests/ipv6_auto/system-connections/eth7.nmconnection

```diff
@@ -9,7 +9,7 @@ interface-name=eth7
 [match]
 
 [ipv4]
-method=auto
```

Comment:
This looks  wrong to me. The corresponding wicked config is:
```
  <ipv4>
    <enabled>true</enabled>
    <arp-verify>true</arp-verify>
  </ipv4>
```

Reply:
Ok, but as long as we do not have an IP setting. No need to set it to `method=auto`?